README.md

```diff
@@ -63,7 +92,35 @@ An example of three clusters configured to use with Submariner would look like t
 
 # Installation
 
-## Setup
+Submariner installation can be done manually using helm charts or by using its operator. Both methods are described here.
+
+## Installation using operator
+
+An operator is a tool to package, manage and deploy any Kubernetes application. A submariner operator will soon be available on [OperatorHub.io](https://operatorhub.io).
+
+Current implementation builds submariner operator and deploys submariner in a [Kind setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md).
+
+### Building the operator
+
+```
+cd operators/go
+make build-operator
+```
+
+### Deploying Submariner using the Operator
+
+After generating the Operator (see docs above), your newly generated operator
+is automatically fully integrated into the Submariner CI automation. Simply use
+the `deploytool` flag to the standard `make` commands.
+
+```make e2e status=keep deploytool=operator```
```

Comment:
The main README shouldn’t mention end-to-end tests, especially since subsequent sections have been removed in favour of the e2e-specific documentation stored elsewhere.

Reply:
+1

Reply:
This "make" is in reference to deploying Subm using operator. Is there any other way?

Reply:
`make e2e status=keep deploytool=operator` doesn’t deploy Submariner using the operator, it runs the end-to-end tests using the Operator to deploy (parts of) Submariner in the test clusters. End users can’t use this to deploy Submariner to an existing Kubernetes cluster.

Documentation on how to deploy using the operator, from an administrator’s perspective, is available in https://docs.google.com/document/d/1G7SCAk0KL2i9Q7hACwG2r_JHL-BhuTcjD04dKDP4shI and would need to be transcribed in the project.

Reply:
As this is a complete rewrite, I will do this as part of a separate PR. The main purpose of this PR was to have correct files and placeholder for the content.